golang/cmd/factoryinsight/http.go

```diff
@@ -44,6 +50,7 @@ func SetupRestAPI(accounts gin.Accounts, version string) {
 		v1.GET("/:customer/:location/:asset", getValuesHandler)
 		v1.GET("/:customer/:location/:asset/:value", getDataHandler)
 	}
+	router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
```

Comment:
There should only be documentation and no changes in the source code

Reply:
Removed line and imports